types/loading.d.ts

```diff
@@ -50,6 +50,8 @@ export interface ElLoading {
 
   /** If you do not have a specific DOM node to attach the Loading directive, or if you simply prefer not to use Loading as a directive, you can call this service with some configs to open a Loading instance. */
   service (options: LoadingServiceOptions): ElLoadingComponent
+
+  directive: object
```

Comment:
there is a `ElLoadingDirective ` interface above at line 37
this line isn't suppose to be `directive: ElLoadingDirective `
i'm not 100% sure tho, but please check

Reply:
and i believe there are more directives missing typings, other than this one, so better check across entire project.